README.md

```diff
@@ -14,8 +14,10 @@ How to set up your GitOps workflow:
 
 5. Add `FLEET_URL` and `FLEET_API_TOKEN` secrets to your new repository's secrets. Learn how [here](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository). Set `FLEET_URL` to your Fleet instance's URL (ex. https://organization.fleet.com). Set `FLEET_API_TOKEN` to an API token for an API-only user in Fleet. Learn how [here](https://fleetdm.com/docs/using-fleet/fleetctl-cli#create-api-only-user).
```

Comment:
Hey @ksatter! Thanks opening this PR.

```suggestion
5. Add the secrets specified at the bottom of [`workflow.yml`](https://github.com/fleetdm/fleet-gitops/blob/main/.github/workflows/workflow.yml) (i.e. `FLEET_URL`) to your new repository's secrets. Learn how [here](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository). Set `FLEET_URL` to your Fleet instance's URL (ex. https://organization.fleet.com). Set `FLEET_API_TOKEN` to an API token for an API-only user in Fleet. Learn how [here](https://fleetdm.com/docs/using-fleet/fleetctl-cli#create-api-only-user).
```

I think we might be starting to duplicate instructions.

What do you think about pointing to `workflow.yml` here in step 5? And then keep step 6 as is.

Does that cover it?